azurelinuxagent/ga/exthandlers.py

```diff
@@ -1117,7 +1113,7 @@ def set_handler_status(self, status="NotReady", message="", code=0):
         except (IOError, ValueError, ProtocolError) as e:
             fileutil.clean_ioerror(e,
                 paths=[status_file])
-            self.logger.error("Failed to save handler status: {0}", traceback.format_exc())
+            self.logger.error("Failed to save handler status: {0}", ustr(e))
```

Comment:
> ustr(e) [](start = 67, length = 8)

The traceback was valuable in locating the cause.  I would not remove it.

Reply:
I removed this because I found that it was not populated, due to the way the threads work. Are you sure this specific one was useful? If so I can add it back.

Reply:
There is no harm in having both - I will add it back along with the error message, in case I missed something.